fv/winfv/policy_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package winfv_test
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+)
+
+func powershell(args ...string) (string, string, error) {
+	ps, err := exec.LookPath("powershell.exe")
+	if err != nil {
+		return "", "", err
+	}
+
+	args = append([]string{"-NoProfile", "-NonInteractive"}, args...)
+	cmd := exec.Command(ps, args...)
+
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	err = cmd.Run()
+	if err != nil {
+		return "", "", err
+	}
+
+	return stdout.String(), stderr.String(), err
+}
+
+func getPodIP(name, namespace string) string {
+	cmd := fmt.Sprintf(`c:\k\kubectl.exe --kubeconfig=c:\k\config get pod %s -n %s -o jsonpath='{.status.podIP}'`,
+		name, namespace)
+	ip, _, err := powershell(cmd)
+	if err != nil {
+		Fail(fmt.Sprintf("could not get pod IP for %v/%v: %v", namespace, name, err))
+	}
+	return ip
+}
+
+func kubectlExec(command string) error {
+	cmd := fmt.Sprintf(`c:\k\kubectl.exe --kubeconfig=c:\k\config -n demo exec %v`, command)
+	_, _, err := powershell(cmd)
+	return err
+}
+
+var _ = Describe("Windows policy test", func() {
+	var (
+		porter, client, clientB, nginx, nginxB string
+	)
+
+	BeforeEach(func() {
```

Comment:
Worth adding comments that pods/policies are created by infra.

Reply:
Done

Reply:
I think more comments about network policies setup are needed so people can understand test cases. For example 
``` 
Three network polices has been installed by test infra
1. Allow cient to nginx 
2
3
````